src/index.html

```diff
@@ -0,0 +1,325 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
У нас русскоязычный сайт

```suggestion
<html lang="ru">
```